pkgs/development/python-modules/s3-credentials/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, click
+, click-default-group
+, boto3
+}:
+
+buildPythonPackage rec {
+  pname = "s3-credentials";
+  version = "0.10";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ieFprsDkxb2w6Qc/DQ58LIfZgZSu/i8ecM1ZnS9hUDs=";
+  };
+
+  propagatedBuildInputs = [
+    click
+    click-default-group
+    boto3
+  ];
+
+  # tests aren't included in the PyPI packages
```

Comment:
Upstream is [tagging](https://github.com/simonw/s3-credentials/releases) the source. Could you check if the tests could be executed by using `fetchFromGitHub` to get the release tarball?

Reply:
sounds good. the tests are now included and run :)